content/vendor/cli/tokens.md

```diff
@@ -6,7 +6,7 @@ weight: 90105
 draft: false
 ---
 
-Using the [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) and [REST API](https://help.replicated.com/api/vendor-api/) requires a token for authorization.  Replicated supports two types of tokens: Team Tokens and User Tokens.
+Using the [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) and [REST API](https://help.replicated.com/api/vendor-api/) requires a token for authorization.  Replicated supports two types of tokens: Service Accounts and User Tokens.
```

Comment:
I don't think we should remove team tokens quite yet. We can take a look at the data from @laverya 's efforts and see what reactions are tomorrow. 
```suggestion
Using the [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) and [REST API](https://help.replicated.com/api/vendor-api/) requires a token for authorization.  Replicated supports three types of tokens: Team , Service Account and User Tokens.
```

Reply:
yeah, i wasn't sure if we should or not and figured someone would complain 😄 i'll add them back in and mention they will be deprecated soon

Reply:
May want to check out the MD I attached; very similar details but the formatting is a bit different. I was unsure about how to allude to the future deprecation at this stage of the game - hopefully there is precedent elsewhere but I haven't looked into it very deeply so far :/

Reply:
done, i ended up moving it to the bottom and taking your suggested changes